smoketest/tests/transfer_token.rs

```diff
@@ -75,11 +80,26 @@ async fn transfer_token() {
         parents: 2,
         interior: Junctions::X2(
             Junction::GlobalConsensus(NetworkId::Ethereum { chain_id: 15 }),
-            Junction::AccountKey20 { network: None, key: DESTINATION_ADDRESS.into() },
+            Junction::AccountKey20 {
+                network: None,
+                key: DESTINATION_ADDRESS.into(),
+            },
+        ),
+    });
+
+    let beneficiary = VersionedMultiLocation::V3(MultiLocation {
+        parents: 2,
+        interior: Junctions::X2(
+            Junction::GlobalConsensus(NetworkId::Ethereum { chain_id: 15 }),
+            Junction::AccountKey20 {
+                network: None,
+                key: DESTINATION_ADDRESS.into(),
+            },
         ),
     });
 
-    let bridge_transfer_call = TransactionApi.transfer_asset_via_bridge(assets, destination);
+    let bridge_transfer_call =
+        TransactionApi.reserve_transfer_assets(destination, beneficiary, assets, 0);
```

Comment:
Seems not correct.  Actually the `0` is the index of fee asset which should be `DOT`, so the assets should be an array which contains both the transfer asset and fee asset.

Reply:
Yep, `transfer_token` [doesn't work yet](https://github.com/Snowfork/snowbridge/pull/972#issuecomment-1778921269). @vgeddes mentioned we can fix it after merge, since it is dependent on https://github.com/paritytech/polkadot-sdk/pull/1672 too.